app/api/views/key_values.py

```diff
@@ -84,4 +84,9 @@ def upsert(key):
         return jsonify(saved), 200
 
     except Exception as error:
-        return abort(error.message)
+        try:
+            msg = error.message
+        except AttributeError:
+            msg = str(error)
+
+        return abort(msg)
```

Comment:
avoid throwing exception, return abort(str(error))

Reply:
This looks fine to me for now.  We can think about changing it in the future.